core/src/main/java/com/homeaway/streamplatform/streamregistry/StreamRegistryApplication.java

```diff
@@ -155,9 +157,30 @@ public void run(final StreamRegistryConfiguration configuration, final Environme
 
         registerManagedContainer(environment, managedKStreams, managedInfraManager, managedKafkaProducer);
 
+        // Step 7 - register list of Jersey Filters.
+        registerFilters(environment, configuration);
+
         // TODO: Make project completely based on unit tests (integration should be a separate project) (#100)
     }
 
+    private void registerFilters(Environment environment, StreamRegistryConfiguration configuration) {
+        try {
+            if (configuration.getRequestFilterClassNames() != null) {
+                for (String requestFilterClassName : configuration.getRequestFilterClassNames()) {
+                    environment.jersey().register(Utils.newInstance(requestFilterClassName, ContainerRequestFilter.class));
+                }
+            }
+
+            if (configuration.getResponseFilterClassNames() != null) {
+                for (String responseFilterClassName : configuration.getResponseFilterClassNames()) {
+                    environment.jersey().register(Utils.newInstance(responseFilterClassName, ContainerResponseFilter.class));
```

Comment:
Consider adding a `log.info("Registering %s ResponseFilter",...);`

Reply:
fixed it